src/presentation/pages/Note.vue

```diff
@@ -40,7 +41,7 @@ const props = defineProps<{
   /**
    * Parent note id, null for root note
    */
-  parentId: string | null;
+  parentId?: string | null;
```

Comment:
describe null/undefined difference meaning

Reply:
null is not needed, removed